code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -73,8 +73,8 @@
 	list_reagents = list("chloralhydrate" = 15)
 
 /obj/item/reagent_containers/glass/bottle/charcoal
-	name = "antitoxin bottle"
-	desc = "A small bottle of charcoal."
+	name = "charcoal (antitoxin) bottle"
```

Comment:
leaving the (antitoxin) makes it a bit confusing imo

Reply:
Yeah, basically every player knows what charcoal does and the new ones will learn

Reply:
I'll gut the antitoxin part then